Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -106,3 +106,31 @@
     - Write
   - type: DisarmMalus
     malus: 0
+
+- type: entity
+  name: energy cutlass
+  parent: EnergySword
+  id: EnergyCutlass
+  suffix: E-Cutlass
+  description: An exotic energy weapon.
+  components:
+  - type: EnergySword
+    secret: true
+    litDamageBonus:
+        types:
+            Slash: 10.5
+            Heat: 10.5
+            Blunt: -1
```

Comment:
This is just a straight upgrade over the regular esword. Maybe lower the attack speed to balance it?

Reply:
cutlass does 16 dam and the e-cutlass same as the e-sword